src/hypergraph.jl

```diff
@@ -309,11 +309,11 @@ function remove_hyperedge!(h::Hypergraph, e::Int)
 end
 
 """
-	clean!(h::Hypergraph)
-Removes all verticies with degree 0 and all hyperedges with size 0.
+    prune_hypergraph!(h::Hypergraph)
+Removes all vertices with degree 0 and all hyperedges of size 0.
 """
 
-function clean!(h)
+function prune_hypergraph!(h)
```

Comment:
Should be:
```julia
function prune_hypergraph!(h::Hypergraph)
```
since this function only makes sense only for a `Hypergraph` object this increases readability of the code.

Reply:
did